WcaOnRails/app/helpers/application_helper.rb

```diff
@@ -124,6 +124,8 @@ def process_nav_items(nav_items)
 
   def wca_date_range(from_date, to_date, options={})
     options[:separator] = '-'
-    date_range(from_date, to_date, options)
+    if from_date && to_date
```

Comment:
I'd rather this method blow up if `from_date` or `to_date` is nil, and force the people calling this method to handle that case explicitly.